src/containers/User/Login.tsx

```diff
@@ -1,66 +1,135 @@
+      }
     }
   }
 
-  if (isLoggedIn()) push('/')
+  const finishLogin = async (loginPayload: LoginPayload) => {
+    const { JWT, user, templatePermissions } = loginPayload
+    await onLogin(JWT, user, templatePermissions)
+    if (history.location?.state?.from) push(history.location.state.from)
+    else push('/')
+  }
+
+  useEffect(() => {
+    if (orgList.length === 0) {
+      // No orgs, so skip org login
+      JWT && user && templatePermissions && finishLogin({ JWT, user, templatePermissions })
+      return
+    }
+    if (orgList.length === 1) {
+      // Auto-login with only one organisation
+      handleSubmit()
+    }
+  }, [orgList])
+
+  const handleSelection = (e: any, data: any) => {
+    const { value } = data
+    setSelectedOrgIndex(value)
+  }
 
   return (
     <Container text style={{ height: '100vh' }}>
       <Grid container columns="1" centered verticalAlign="middle" style={{ height: '100%' }}>
-        <Grid.Column>
-          <Segment>
+        <Grid.Column width={14} style={{ padding: 10 }}>
+          <Segment clearing>
             <Header size="huge">{strings.LABEL_WELCOME}</Header>
             <Form>
-              <Form.Field>
-                <label>{strings.LABEL_LOGIN_USERNAME}</label>
-                <input
-                  placeholder="Username"
-                  name="username"
-                  type="text"
-                  value={username}
-                  onChange={(event) => setUsername(event.target.value)}
-                />
-              </Form.Field>
-              <Form.Field>
-                <label>{strings.LABEL_LOGIN_PASSWORD}</label>
-                <input
-                  placeholder="Password"
-                  name="password"
-                  type="password"
-                  value={password}
-                  onChange={(event) => setPassword(event.target.value)}
+              {!user && (
+                <>
+                  <Form.Field>
+                    <label>{strings.LABEL_LOGIN_USERNAME}</label>
+                    <input
+                      placeholder="Username"
+                      name="username"
+                      type="text"
+                      value={username}
+                      onChange={(event) => setUsername(event.target.value)}
+                    />
+                  </Form.Field>
+                  <Form.Field>
+                    <label>{strings.LABEL_LOGIN_PASSWORD}</label>
+                    <input
+                      placeholder="Password"
+                      name="password"
+                      type="password"
+                      value={password}
+                      onChange={(event) => setPassword(event.target.value)}
+                    />
+                  </Form.Field>
+                </>
+              )}
+              {user && (
+                // To-Do: Add to Messages
+                <p>{`Welcome back, ${user.firstName}. Please select your organisation.`}</p>
```

Comment:
Also can this one be done already please? I think it is best to keep all these strings in a file already so we know all that we need to localise when the time comes